lib/index.js

```diff
@@ -1,8 +1,8 @@
 const lex = require('./lexer')
 const parse = require('./parser')
 
-module.exports = function SugarMLParser (input) {
-  return parse(lex(input))
+module.exports = function SugarMLParser (input, options) {
+  return parse(lex(input, options))
```

Comment:
This options object is the parser options, not reshape's options. That would still need to be PR'd